js/client-teambuilder.js

```diff
@@ -1330,6 +1330,24 @@
 			this.curChartName = '';
 			this.update();
 			this.$('input[name=pokemon]').select();
+			if (this.curTeam.format.includes('monotype')) {
+				var typeTable = [];
+				for (var i = 0; i < this.curSetList.length; i++) {
+					const species = Dex.forGen(this.curTeam.gen).species.get(this.curSetList[i].species);
```

Comment:
I believe the client is still using getSpecies etc outside of build files currently

Reply:
> I believe the client is still using getSpecies etc outside of build files currently

it was matched to update the server api here https://github.com/smogon/pokemon-showdown-client/commit/31fcd418ecc905a4b46ba1f1dda9d19cb7755b4e

Reply:
oh its because ur using `const` then mb